bindings/wysiwyg-ffi/README.md

```diff
@@ -12,9 +12,18 @@ wysiwyg-rust.
   android-ndk-r22b-linux-x86_64.zip and unzip it e.g.:
 
 ```bash
-unzip android-ndk-r22b-linux-x86_64.zip
+cd $ANDROID_SDK_HOME
```

Comment:
It seems like there are several variables that can be used here:

- ANDROID_SDK_ROOT
- ANDROID_SDK_HOME
- ANDROID_HOME

It seems like the proper one is `ANDROID_HOME`, the other 2 are deprecated. There is also `ANDROID_NDK_ROOT` that we could have tried before. It's probably worth to mention that this variable should be set somewhere before we try to use it here.